src/common/models/sort-on/sort-on.mocha.ts

```diff
@@ -10,6 +10,33 @@ describe('SortOn', () => {
   it('is an immutable class', () => {
     testImmutableClass(SortOn, [
 
+      {
+        measure: {
+          name: 'price',
+          title: 'Price',
+          expression: $('main').min('$price').toJS()
+        }
+      },
+      {
+        measure: {
+          expression: {
+            action: {
+              action: 'sum',
+              expression: {
+                name: 'price',
+                op: 'ref'
+              }
+            },
+            expression: {
+              name: 'main',
+              op: 'ref'
+            },
+            op: 'chain'
+          },
+          name: 'price',
+          title: 'Price'
+        }
+      }
```

Comment:
one possible sortOn is where there is only a dimension. You are only testing the measure case. Could you please add a dimension test case as well.